### ISSUE TITLE
Missing conda package for ivar_variants_to_vcf.nf

### ISSUE THREAD

**Issue by wutron** [nf-core/viralrecon#369]
### Description of the bug

`ivar_variants_to_vcf.py` has `from Bio import SeqIO`, which means that `ivar_variants_to_vcf.nf` requires biopython (e.g. `conda-forge::biopython=1.81`).

### Command used and terminal output

```console
Traceback (most recent call last):
  File "ivar_variants_to_vcf.py", line 12, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'
```

### System information

Nextflow version: 19.01.0.5050
Hardware: Cloud VM
Executor: local
Container engine: Conda
OS: Linux
Version of nf-core/viralrecon: 2.5

**Comment by wutron** [comment]
Oh, already fixed in 2f57860. Great!

**Comment by drpatelh** [comment]
Beat me to it!
